src/containers/Login.vue

```diff
@@ -115,58 +99,23 @@
       </div>
 
       <!-- SEMESTERS -->
-      <div class="container-fluid p-0 small">
-        <div class="row semester phonepad m-0">
-          <div class="col-12 col-md-5 comment">
+      <div class="container-fluid p-0 center">
+        <div class="semester phonepad row no-gutters float-right">
```

Comment:
flexbox is generally considered the better way to do something like this. Look up `justify-content: flex-end`.